### ISSUE TITLE
nixops fails to deploy for even trivial nix files (`error: execv() arg 2 must contain only strings`)

### ISSUE THREAD

**Issue by maurer** [NixOS/nixops#338]
```
[maurer@fragarach:~/Development/nixops]$ git log --pretty=format:'%H' -n 1
6bb81897e97b3904fbcf08428f22f0f6ca919a02

[maurer@fragarach:~/Development/nixops]$ ./dev-shell 

[nix-shell:~/Development/nixops]$ nixops create examples/trivial-container.nix
created deployment ‘307fbac5-5bf5-11e5-92d0-56847afe9799’
307fbac5-5bf5-11e5-92d0-56847afe9799

[nix-shell:~/Development/nixops]$ nixops info
error: execv() arg 2 must contain only strings

[nix-shell:~/Development/nixops]$ nixops deploy
error: execv() arg 2 must contain only strings

[nix-shell:~/Development/nixops]$ 
```

**Comment by domenkozar** [comment]
What version of nixops are you using?

**Comment by maurer** [comment]
Git master, thus why I included git log output

**Comment by rbvermaa** [comment]
Fixed in da079b2be87a78523665ee447d98ca8c6947217d
